ui/ducks/send/helpers.js

```diff
@@ -72,30 +59,21 @@ export async function estimateGasLimitForSend({
       toAddress: to,
       amount: value,
     });
-
-    paramsForGasEstimate.to = sendToken.address;
   } else {
     if (!data) {
       // eth.getCode will return the compiled smart contract code at the
       // address. If this returns 0x, 0x0 or a nullish value then the address
-      // is an externally owned account (NOT a contract account). For these
-      // types of transactions the gasLimit will always be 21,000 or 0x5208
+      // is an externally owned account (NOT a contract account).
       const { isContractAddress } = to
         ? await readAddressAsContract(global.eth, to)
         : {};
-      if (!isContractAddress && !isNonStandardEthChain) {
-        return GAS_LIMITS.SIMPLE;
```

Comment:
In the long run, I don't think we want these constants to be in place rather than asking `estimateGas` in the controller API.